roles/testnode/defaults/main.yml

```diff
@@ -29,8 +29,14 @@ epel_packages: []
 # version, distro or package type
 common_packages: []
 
+# common packages that aren't available in aarch64 architecture
+non_aarch64_packages: []
+non_aarch64_packages_to_upgrade: []
+non_aarch64_common_packages: []
+
 # packages used by ceph we want to ensure are removed
 ceph_packages_to_remove: []
+ceph_dependency_packages_to_remove: []
```

Comment:
`roles/testnode/vars/yum_systems.yml` isn't included on playbook runs where `ansible_pkg_mgr != apt`.  Even though the tasks relying on those vars are skipped, the tasks were still fatally failing because the vars weren't defined.  Defining empty defaults gets around that.

Not arguing the confusing aspect.  Just explaining why it also needs to be set here.  Otherwise we need to use `|default([])`.